styleguide/writing-style.md

```diff
@@ -17,14 +17,14 @@ This section covers everything related to the language and formatting used in Bu
 
 ### English flavor
 
-Use US English. As a highly multi-national team, here is the list of the most notable [differences between American, British, and Australian English](http://web.archive.org/web/20180925084522/http://www.linktranslation.com.au/the-differences-between-american-british-and-australian-english/) English to watch out for.
+Use US English. As a highly multi-national team, here is the list of the most notable differences between [Australian, American (US), and British English](https://blog.e2language.com/australian-english-vs-american-english-vs-british-english/) English to watch out for.
```

Comment:
I thought it might be more appropriate to update this external reference to one which is no longer archived (and hence may disappear at any time, a bit like deprecated code).

I searched around for a couple of alternatives and this one seemed most appropriate.